doc/development/branching_how-to.md

```diff
@@ -0,0 +1,130 @@
+# Branching How-To
+
+## Assumptions
+
+Given the creation of a new release branch will typically happen right
+before RC1 of a new release series, please see assumptions in the
+`howto_release.md` document.
+
+## Create a New Branch
+
+Given how the branch protection rules are set up and work,
+you need to bypass protection against merge commits on branches.
+(We don't want any new merge commits. However, there are merge commits
+from the past and they prevent creation of a new branch when the rules are
+applied.)
```

Comment:
```suggestion
We don't want any new merge commits. However, there are merge commits
from the past and they prevent the creation of a new branch when the rules are
applied.
```

Reply:
It is a parenthetical statement, so I used parenthesis.